app/controllers/topics-controller.js

```diff
@@ -1,8 +1,8 @@
 const {fetchTopics} = require('../models/topics-model')
 
 exports.getTopics = (req, res, next) => {
-    fetchTopics().then(({rows}) => {
-        res.status(200).send(rows)
+    fetchTopics().then((topics) => {
+        res.status(200).send(topics)
```

Comment:
This should be sending back an object with the key of topics and the value of an array of topics.